### DISCUSSION TITLE
Make default color palettes go to darker tones

### DISCUSSION THREAD

**Discussion by nicoszerman** [tailwindlabs/tailwindcss#7988] [Ideas]
I am developing the website for my startup and I was trying to set a dark background color with contrasting light (similar to [stripe.com](https://stripe.com/) if you scroll to "Designed for developers"). However, after trial and error it was sad for me to find out that the darkest tones of the included color palettes are actually pretty light. This is hard to notice in the official website because the lighter colors to the left serve as contrast points, facilitating an optical illusion of dark tone, but if you look at the darkest tones of red, green, blue, etc.. you will notice they are not so dark. Except for the gray-scales (slate, zinc, neutral, stone), the 900s are too light to be able to use contrasting light text against them.

How about adding sky-1000, rose-1000, or a set of dark tones maybe dark-sky, dark-rose, etc.. ?

In particular, adding sky-1000, sky-1100, etc.. looks like a very non-intrusive addition that facilitates what seems to me an expected feature: dark color tones.

**Comment by kerryj89** [comment]
I agree, the neutral/muted color's 950 tend to work well as a background color, however, the non-neutral colors seem to stop short.  I ended up creating a 1000 scale for the color as a background to compliment the neutral/muted 950's.  They are dark but have a noticeable tint to them when used in large areas like the background.
